pandas/io/json/_json.py

```diff
@@ -347,6 +347,9 @@ def _write(
 
 
 @deprecate_kwarg(old_arg_name="numpy", new_arg_name=None)
+@deprecate_nonkeyword_arguments(
+    version="1.4", allowed_args=["path_or_buf"], stacklevel=3
```

Comment:
The reason for the stacklevel=3 is because there is another decorator around it?

Reply:
Yes, it's looking like greater stack level is needed in case of multiple decorators.